packages/client/TODO.md

```diff
@@ -16,7 +16,7 @@
 
 Client manages the token lifecycle and storage
 
-- LensClient.init({ environment, storage })
+- new LensClient({ environment, storage? })
```

Comment:
we decided that there is no need for a static constructor and `new` is a more obvious way